app/assets/javascripts/academiesUtils.js

```diff
@@ -188,9 +188,24 @@ const formatAcademyRowsVersion5Ofsted = (academies) =>
         text: academy.localAuthority,
       },
       {
-        html: `${academy.phase}<br>${academy.minPupilAge} - ${academy.maxPupilAge}`,
+        text: academy.type,
+      },
+      {
+        html: `<p>Urban city and town</p>`,  
+      },
+      {
+        html: `<p><a href="./#">Get information about schools</a></p>`,  
```

Comment:
```suggestion
        html: `<p><a href="#">Get information about schools</a></p>`,  
```

This link tries to navigate to 'version-5/#' with an application error, If you remove the dot (href="#") it will just reload the page (function as fake link)